index.js

```diff
@@ -44,6 +44,10 @@ var CqlshDriver = Base.extend({
     metaData.push('PRIMARY KEY ' + primaryKey);
     cql.push(metaData.join(', '));
     cql.push(')');
+    if (constraints.compression) {
+      cql.push('WITH compression =');
+      cql.push(constraints.compression);
+    }
```

Comment:
There is a separate pull request for this change, can you remove this from this pull request.